pkg/mcs/scheduling/server/config/config.go

```diff
@@ -299,9 +331,14 @@ func (o *PersistConfig) IsOneWayMergeEnabled() bool {
 	return o.GetScheduleConfig().EnableOneWayMerge
 }
 
-// GetMergeScheduleLimit returns the merge schedule limit.
-func (o *PersistConfig) GetMergeScheduleLimit() uint64 {
-	return o.GetScheduleConfig().MergeScheduleLimit
+// GetMaxMergeRegionSize returns the max region size.
```

Comment:
In future, we will need to add two in `pkg/mcs/scheduling/server/config/config.go` and `server/config/persist_options.go`?

Reply:
Yes